src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+
+4. Choose your installation method:
+
+<!--
+Guideline: Add SDK specific installation information
+-->
+
+## Initialize {#initialize}
+
+`init` Sentry as soon as possible:
+<!--
+Guideline: Add init code example for this SDK 
+-->
+Once this is done, all unhandled exceptions are automatically captured by Sentry. 
+
+**Important: Record your DSN.** The *DSN* (Data Source Name) tells the SDK where to send the error, associating errors with the project you just created. If you forget your DSN, log in to Sentry to obtain the project DSN from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI. The DSN includes the protocol, public key, server address, and project identifier, using this format:
+
+`https://<Public Key>@<Sentry Server Address>/<Project Identifier>`
+
+## Verify Setup {#verify-setup}
+
+Verify your setup by intentionally sending an error that breaks your application. Sentry will send an alert to the email address configured on [Sentry.io](http://sentry.io/) notifying you about the event.
+
+An *event* is one instance of sending data to Sentry. Generally, this data is an error. An *issue* is a grouping of similar events.
+
+For example, in <!--SDK name-->, 
+<!--
+Guideline: Add verify setup example for the SDK you are documenting
+-->
+Resolve the created error by either: 
+
+- Clicking `View on Sentry` in the alert sent to the email provided when you established your account
+- Logging in to Sentry, then opening your project
+
+On the **Issue Details** page, scroll down to the Exception stack trace and view the error. 
+
+## Capture Errors {#capture-errors}
+
+The reporting of an *event* - an error, exception, or rejection - is called *capturing*. When an event is captured, it’s sent to Sentry, then creates a new issue group (or is added to an already existing one) based on Sentry’s grouping algorithm. 
+<!--
+Guideline: Add SDK specific information, as appropriate
+-->
+Learn more about how to manually capture errors or enable message capture with the <!--SDK name--> SDK in Capture Errors <!--link to the Capture Errors content for this SDK-->.
+
+### Enrich Error Data {#enrich-error-data}
+
+When the SDK sends an event to Sentry, the event is enriched with data to help identify the source of the event. We include data both pertinent to the event as well as what led up to the event. Much of this data is sent automatically - including the error context <!--link to the Event Context content for this SDK--> and environment <!--link to the Manage Environments--> - as well as the trail of events that happened prior to an issue, which we call breadcrumbs <!--link to the breadcrumbs content for this SDK-->. You don't need to configure these, though you may modify them. 
+
+Learn more about enriching data sent with events in Enrich Event Data <!--link to the Enrich Event Data page for this SDK-->.
```

Comment:
i would link to every kind of information they can add, because they may already have an idea for what they're looking for.

Reply:
The Enrich Error Event page has all the types listed, so I thought it more efficient to add a single link, in case it's confusing/overwhelming - 
Event Context
Breadcrumbs
Manage Environments
Track Releases
Add User Feedback